.travis.yml

```diff
@@ -1,8 +1,18 @@
-language: node_js
 sudo: true
+
+language: node_js
+
 node_js:
-  - "0.10"
-  - "0.12"
-  - "iojs"
-  - "4"
-  - "5"
+  - 6
+  - 5
+  - 4
+  - 0.12
+  - 0.10
```

Comment:
these MUST be surrounded in quotes, or else travis will interpret this as "0.1".